### PR TITLE
Uses the width of the terminal to grow the table width as needed.

### PR DESCRIPTION
**Pull request by gabrielcsapo**
*Why?*

This will allow developers to see broccoli funnels names if they simply expand their terminal width.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,16 +8,20 @@ function ellipsize(string, desiredLength) {
...
     var summary = calculateSummary(tree);
     var pcThreshold = factor || 0.05;
     var msThreshold = pcThreshold * summary.totalTime;
     var cumulativeLogLines = [];
 
-    var MAX_NAME_CELL_LENGTH = 45;
-    var MAX_VALUE_CELL_LENGTH = 20;
 
+    var MAX_TABLE_LENGTH = process.stdout.columns || 100;
+    // the name column is 8/10th the size of the value celll
```
Comment (rwjblue) [inline index.js:21]:
```suggestion
    // the name column is 8/10th the size of the value cell
```

**Review by rwjblue — COMMENTED** [review]
What I'd like us to do is to use the smallest space needed for the table **but** allow it to stretch all the way to the full terminal width, do you think that is possible?

**Comment by gabrielcsapo** [comment]
@rwjblue updated based on your suggestion

**Review by rwjblue — APPROVED** [review]
Awesome!!! Thank you @gabrielcsapo!

File: index.js [hunk 2]
```diff
@@ -8,16 +8,33 @@ function ellipsize(string, desiredLength) {
   }
 }
 
-module.exports = function printSlowNodes(tree, factor) {
+function getLargestString(strings) {
+  return Math.max(...(strings.map(string => string.length)));
```
Comment (rwjblue) [inline index.js:12]: 😭 we haven't dropped old node (still support Node 0.12 here)

CI fails here:

```
/home/travis/build/rwjblue/broccoli-slow-trees/index.js:12
  return Math.max(...(strings.map(string => string.length)));
                  ^
SyntaxError: Unexpected token .
```

```suggestion
  return Math.max.apply(Math, strings.map(string => string.length));
```

**Comment by rwjblue** [comment]
Released in v3.1.0

**Pull request merged**